### ISSUE TITLE
Doesn't work with Kodi

### ISSUE THREAD

**Issue by 5hay** [bl4ckic3/IranProudCrawler#1]
Hi,

I created the test.txt with "python2.7 crawler.py" and tried opening it with Kodi's IPTV Simple Client, but it doesn't work.

Using Kodi 14.2 (even tried ne newest 15.x version) on Mac OS X 10.10.5

**Comment by 5hay** [comment]
Changed the output file a lil bit, now it works flawlessly.

```
#EXTM3U

#EXTINF:-1, VARZESH_TV
http://63.237.48.23/ios/VARZESH_TV/VARZESH_TV.m3u8
#EXTINF:-1, ICC
http://63.237.48.23/ios/ICC/ICC.m3u8
#EXTINF:-1, GEM_TV
http://63.237.48.23/ios/GEM_TV/GEM_TV.m3u8
```

The `#EXTM3U` in the first line is neccessary for it to work.

**Comment by AlixAbbasi** [comment]
Fixed in version 1.2. Please let me know if you find any error in v 1.2
